### ISSUE TITLE
WCAG 2.2 Violation - There is text with insufficient contrast present in the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#1]
### Issue
There is text present on the page that does not meet the minimum required contrast ratio with its background (4.500:1 for most text; 3.000:1 for text that is 24px or larger).

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG22/#contrast-minimum)

### Instances

1. #20 
2. #21 
3. #22 
4. #23 
5. #57 
6. #58 
7. #54 
8. #55 
9. #56 ### User Impact

Some users (such as users with low vision, or users who are outdoors and experiencing glare on their screens) may be unable to read the text content.

#### Further information:

1. ["Web Accessibility Perspectives: Colors with Good Contrast" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=Hui87z2Vx8o&list=PLhDEeYUfW02QndusXXtQtuMbMYhK7TMBT&index=2)
2. ["Web Accessibility Perspectives: Colors with Good Contrast - Audio Described Version" by W3C Web Accessibility Initiative (WAI) (YouTube) (links to the video in a broader playlist context)](https://www.youtube.com/watch?v=a9kNUv6N8Rk&list=PLhDEeYUfW02Qo4r2KlzagxZxhYcZADee-&index=9)
3. [The "Intent" section of "Understanding Success Criterion 1.4.3: Contrast (Minimum) | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#intent)

### Steps To Reproduce

Using a colour contrast checker tool such as [Accessible Color Picker](https://chromewebstore.google.com/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld):

1. Check the final, rendered ("computed") font size of the text using your browser's developer tools.
2. Use the colour contrast checker tool's eyedropper tool to grab the final, rendered text colour value, as well as the final, rendered background colour value, and calculate the contrast ratio between the text colour ("foreground") and background colour ("background").
3. Notice that the text does not meet the minimum required contrast ratio (4.500:1 for most text; 3.000:1 for text that is 24px or larger).

### How To Fix The Issue

1. Update the text colour, the background colour, or both to meet (or exceed) the minimum required contrast ratio (4.500:1 for most text; 3.000:1 for text that is 24px or larger).
2. Repeat the "Steps To Reproduce" outlined above, verifying that the text now meets the minimum required contrast ratio.

#### WCAG 2.2 Conformant Code Example

Not available/not applicable.
